### ISSUE TITLE
fix(ux): selecting all entities is not possible in 'create a relationship'

### ISSUE THREAD

**Issue by ghost** [LuatixHQ/opencti#6625]
## Description
Selecting all queried entities is not possible in the knowledge section of a entity

## Environment

1. OS (where OpenCTI server runs): debian
2. OpenCTI version: 6.0.5
3. OpenCTI client: web
4. Other environment details:

## Reproducible Steps
1. Go to the knowledge tab of a vulnerability
2. Click Relationships View
3. Click the + Button
4. Search for the knowledge you want to add
5. See that the Select all checkbox is disabled

## Expected Output
Select all checkbox is enabled and allows me to select all results of my query

## Actual Output
Select all checkbox is disabled

## Screenshots (optional)
![Screenshot from 2024-04-08 12-05-56](https://github.com/OpenCTI-Platform/opencti/assets/36128722/1f4e015c-09ce-4748-ad34-db7c92d98ad3)

**Comment by nino-filigran** [comment]
@iFrozenPhoenix This is not a bug actually: we've done this on purpose to prevent a user creating an enormous amount of relations with a single entity (which would create performance issues), since relation are created on the go.
However, I do agree that this design is not the best: indeed, if a user has done (like you) the effort of looking & filtering the list, resulting in less entities (or at least a manageable amount) we should allow the select-all. I've transformed your ticket into a feature.

**Comment by snitram59** [comment]
@nino-filigran would it be possible to prioritize this one ?

**Comment by nino-filigran** [comment]
No, it was never possible before (I'm going to change it back to a feature), and we have other feature with higer priority at the moment. So it won't be prioritzed. You can do shift+click in the meantime.
